src/OpenTracing/Tracer.php

```diff
@@ -25,6 +25,13 @@
     const FORMAT_HTTP_HEADERS = 3;
 
     /**
+     * @deprecated use either startActiveSpan or startManualSpan instead.
```

Comment:
The problem is that we'd prefer the python one to be released before merging this one so either we wait or either we release the library without the active span record.